docs/install.md

```diff
@@ -81,6 +81,11 @@ Check spack is in `$PATH`, for example
 spack --version
 ```
 
+### Version
+
+We recommend using spack v0.21.0 or later. Older versions of spack may work but are not 
```

Comment:
```suggestion
We recommend downloading locally and using the latest spack version (develop). If a stable version is needed, that has to be spack v0.21.0 or later at the time of writing. Older versions of spack may work but are not 
```

Reply:
I think the downloading part is covered earlier. Agree about recommending the latest. How about

https://github.com/ukri-excalibur/excalibur-tests/blob/170aef375bfd6f4d157d9e42c09d32f012164abf/docs/install.md?plain=1#L86-L87